src/docs/cookbook/design/orientation.md

```diff
@@ -12,27 +12,18 @@ js:
     url: https://dartpad.dev/inject_embed.dart.js
 ---
 
-In some situations,
-you want to update the display of an app when the user
-rotates the screen from portrait mode to landscape mode. For example,
-the app might show one item after the next in portrait mode,
-yet put those same items side-by-side in landscape mode.
+Trong vài trường hợp,
+bạn muốn thay đổi hiện thị của ứng dụng khi người dùng xoay màn hình từ chế độ chân dung sang chế độ phong cảnh. Ví dụ, ứng dụng sẽ hiển thị một item, trượt qua để thấy item tiếp theo, còn với chế độ phong cảnh, các item sẽ hiển thị ngang.
```

Comment:
"_Trong vài trường hợp bạn có thể muốn thay đổi_ hiển thị của ứng dụng khi người dùng xoay màn hình từ chế độ chân dung sang chế độ phong cảnh. _Ví dụ, ứng dụng có thể hiện thị các item dạng danh sách dọc ở chế độ chân dung nhưng hiển thị các item cạnh nhau phương ngang ở chế độ phong cảnh._"

Reply:
In Vietnamese, "màn hình ngang or chế độ ngang" and "màn hình dọc và chế độ màn hình dọc" are more similar than "chế độ chân dung" and "chế độ phong cảnh".

Reply:
In Vietnamese, "màn hình ngang or chế độ ngang" and "màn hình dọc và chế độ màn hình dọc" are easier to understand than "chế độ chân dung" and "chế độ phong cảnh".